docs/wiki/EL/el-specs.md

```diff
@@ -0,0 +1,1335 @@
+2. **Excess Blob Gas Validation**: Calculate excess blob gas from the parent header and ensure it matches the current blocks header parameter excess_blob_gas
+2. **Header Validation**: Compare and validate the current block's header against that of the parent block.
+3. **Ommers Field Check**: Verify that the ommers field in the current block is empty. Note: "ommers" is the gender-neutral term that replaces the previously used term "uncles."
+4. **Block Execution**: Execute the transactions within the block, which yields the following outputs:
+   - **Gas Used**: The total gas consumed by executing all transactions in the block.
+   - **Trie Roots**: The roots of the tries for all transactions and receipts contained in the block.
+   - **Logs Bloom**: A bloom filter of logs from all transactions within the block.
+   - **State**: The state, as specified in the python execution specs, after executing all transactions.
+5. **Header Parameters Verification**: Confirm that the parameters returned from executing the block are present in the block header. This includes comparing the state's root with the `state_root` field in the block header.
+6. **Block Addition**: If all checks are successful, append the block to the blockchain.
+7. **Pruning Old Blocks**: Remove blocks that are older than the most recent 255 blocks from the blockchain.
+8. **Error Handling**: If any validation checks fail, raise an "Invalid Block" error. Otherwise, return None.
+
+Client Code :
+
+| function | EELS(cancun) | Geth | Reth | Erigon | Nethermind | Besu |
+|----------|------|------|--------|------------|------|-----|
+| Block Level  STF |    | Header verification begins here:  [insertChain]( https://github.com/ethereum/go-ethereum/blob/38eb8b3e20bf237a78fa57e84fa63c2d05a44635/core/blockchain.go#L1628) -> Transaction Iteration : [ Process ](https://github.com/ethereum/go-ethereum/blob/6f1fb0c29ff25318e688c15581d0c28dcefb75ce/core/state_processor.go#L60) -> txLevel STF    |      |        |            |      |
+
+# Block Header Validation
+
+The process of block header validation, rigorously defined within the Yellow Paper + Spec, encompasses extensive checks, including gas usage, timestamp accuracy, among others. This meticulous validation process ensures every block strictly complies with Ethereum's stringent standards, thereby upholding the network's security and operational stability. Furthermore, it delineates the boundaries of Ethereum's economic model, integrating essential safeguards against inefficiencies and vulnerabilities. Through this process, the Ethereum network achieves a delicate balance between flexibility, allowing for evolution and upgrades, and rigidity, ensuring adherence to core principles and safety measures.
+
+ 
+The Ethereum economic model, as outlined in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559), introduces a series of mechanisms aimed at enhancing network efficiency and stability:
```

Comment:
1559 is not the 'Ethereum economic model', that sounds quite confusing. It's transaction fee market mechanism. Also, this feel pretty off topic in this section and I am not sure if this big part of EELS doc should be dedicated just to 1559 motivation. If you wish to keep this, I will post some more comments on this